src/app/zap-templates/zcl/data-model/chip/replacable-monitoring-cluster.xml

```diff
@@ -0,0 +1,324 @@
+      <description>Reset the condition of the replaceable to the non degraded state</description>
+    </command>
+  </cluster>
+
+  <cluster>
+    <name>Water Tank Monitoring</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>Attributes and commands for monitoring water tanks in a device</description>
+    <code>0x0079</code>
+    <define>WATER_TANK_MONITORING</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+
+    <!-- Attributes -->
+    <attribute side="server" code="0x0000" define="WATER_TANK_MONITORING" type="percent" min="0" max="100" writable="false" isNullable="false">Condition</attribute>
+    <attribute side="server" code="0x0001" define="WATER_TANK_MONITORING" type="DegradationDirectionEnum" min="0" max="1" writable="false" isNullable="false">DegradationDirection</attribute>
+    <attribute side="server" code="0x0002" define="WATER_TANK_MONITORING" type="ChangeIndicationEnum" min="0" max="2" writable="false" isNullable="false" default="0" optional="false">ChangeIndication</attribute>
+    <attribute side="server" code="0x0003" define="WATER_TANK_MONITORING" type="boolean" writable="false" isNullable="false" optional="true">InPlaceIndicator</attribute>
+
+    <!-- Commands -->
+    <command source="client" code="0x00" name="ResetCondition" optional="true">
+      <description>Reset the condition of the replaceable to the non degraded state</description>
+    </command>
+  </cluster>
+
+  <cluster>
+    <name>Fuel Tank Monitoring</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>Attributes and commands for monitoring fuel tanks in a device</description>
+    <code>0x007A</code>
+    <define>FUEL_TANK_MONITORING</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+
+    <!-- Attributes -->
+    <attribute side="server" code="0x0000" define="FUEL_TANK_MONITORING" type="percent" min="0" max="100" writable="false" isNullable="false">Condition</attribute>
+    <attribute side="server" code="0x0001" define="FUEL_TANK_MONITORING" type="DegradationDirectionEnum" min="0" max="1" writable="false" isNullable="false">DegradationDirection</attribute>
+    <attribute side="server" code="0x0002" define="FUEL_TANK_MONITORING" type="ChangeIndicationEnum" min="0" max="2" writable="false" isNullable="false" default="0" optional="false">ChangeIndication</attribute>
+    <attribute side="server" code="0x0003" define="FUEL_TANK_MONITORING" type="boolean" writable="false" isNullable="false" optional="true">InPlaceIndicator</attribute>
+
+    <!-- Commands -->
+    <command source="client" code="0x00" name="ResetCondition" optional="true">
+      <description>Reset the condition of the replaceable to the non degraded state</description>
+    </command>
+  </cluster>
+
+  <cluster>
+    <name>Ink Cartridge Monitoring</name>
```

Comment:
Ideally, all these cluster names would be defined by the spec.  Right now they are not.  The spec never has the string "Ink Cartridge Monitoring" anywhere in it....

Reply:
How would you suggest to do this? The aliases are currently in the table with their cluster ID but it says only `Ink Cartridge`. Should we replace it there with `Ink Cartridge Monitoring`?

Reply:
That was also my initial thinking, that we would replace the column and add in specific cluster names.

In the current format, they would essentially all be called "Replaceable Monitoring" and then the cluster id would indicate what type it is, but that would not be ideal in the XML so I think we should propose this change in the DM TT?

Reply:
Or the table could have both cluster name and what is monitored, but they seem like they would be very redundant.  So yes, I would propose we have the spec have a "cluster name" column.

Reply:
I raised an issue in the spec repo that this does not get lost! (https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6980)